backend/infrahub/menu/menu.py

```diff
@@ -261,32 +261,11 @@ def _extract_node_icon(model: MainSchemaTypes) -> str:
                 namespace="Builtin",
                 name="Webhooks",
                 label="Webhooks",
-                icon=_extract_node_icon(infrahub_schema.get(InfrahubKind.CUSTOMWEBHOOK)),
+                icon=_extract_node_icon(infrahub_schema.get(InfrahubKind.WEBHOOK)),
+                kind=InfrahubKind.WEBHOOK,
```

Comment:
I've updated the menu to only display one entry for Webhooks

Reply:
Did this require a migration?

Reply:
yes but I need to do one for the overall menu redesign so I'll do both at once